### DISCUSSION TITLE
Avira reporting malware threat on the latest UVTools 2.22.0 - cvextern.dll

### DISCUSSION THREAD

**Discussion by cpm64** [sn4k3/UVtools#298] [Q&A] [answered]
Hi,

Just updated to the latest UVTools v2.22 and Avira is reporting:

![s01](https://user-images.githubusercontent.com/53166185/133920093-c0beda3f-7c98-4f24-8da1-d607b17b7748.jpg)

I wonder if this is a false alert or?

Cheers,
CPM

**Comment by sn4k3** [comment, marked as answer]
That's from OpenCV, add UVtools folder or that file in exclusions. Different AVs are reporting different false positives
